modules/lightning_features/lightning_media/src/MediaHelper.php

```diff
@@ -37,17 +37,26 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager) {
    *   (optional) Whether to filter the bundles by create access for the current
    *   user. Defaults to FALSE.
    *
+   * @param array $target_bundles
```

Comment:
Nit -- there should not be a blank line above this parameter. Also, the type hint should be string[].